src/shared/bus-unit-util.c

```diff
@@ -1123,7 +1123,8 @@ static int dump_processes(
                 assert(n == cg->n_children);
                 qsort_safe(children, n, sizeof(struct CGroupInfo*), cgroup_info_compare_func);
 
-                n_columns = MAX(LESS_BY(n_columns, 2U), 20U);
+                if (n_columns != 0)
+                    n_columns = MAX(LESS_BY(n_columns, 2U), 20U);
```

Comment:
Patch looks good, but could you please follow the usual indentation? (i.e. multiples of 8ch, like everywhere else, also see CODING_STYLE). Thanks!